### PR TITLE
Writing a Dev Service doc fixes

### PR DESCRIPTION
**Pull request by gastaldi**
This removes the extra brace (`{`) in the snippet

### CODE REVIEW

**Review by holly-cummins — APPROVED** [review]

**Comment by holly-cummins** [comment]
Thanks!

**Comment by gastaldi** [comment]
Hm, I wonder why you don't have the karma to make PR mergeable on approval. Perhaps @gsmet may help

**Review by geoand — APPROVED** [review]

File: docs/src/main/asciidoc/extension-writing-dev-service.adoc [hunk 1]
```diff
@@ -23,11 +23,11 @@ include::_attributes.adoc[]
 If your extension provides APIs for connecting to an external service, it's a good idea to provide a xref:dev-services.adoc[Dev Service] implementation.
 
 To create a Dev Service, add a new build step into the extension processor class that returns a `DevServicesResultBuildItem`.
-Here, the link:https://hub.docker.com/_/hello-world`hello-world` image is used, but you should set up the right image for your service.
+Here, the https://hub.docker.com/_/hello-world[`hello world`] image is used, but you should set up the right image for your service.
```
Comment (gsmet) [inline docs/src/main/asciidoc/extension-writing-dev-service.adoc:26]: Good fix but I think I would keep the name of the image as is.

```suggestion
Here, the https://hub.docker.com/_/hello-world[`hello-world`] image is used, but you should set up the right image for your service.
```

**Review by gsmet — APPROVED** [review]
Nice catch! I made a small suggestion, feel free to ignore if you have a strong opinion :).

**Pull request merged**